File: app/src/main/java/com/example/watering/investrecord/Adapter/List6Adapter.java
package com.example.watering.investrecord.Adapter;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.BaseAdapter;
import android.widget.TextView;

import com.example.watering.investrecord.Info.Info_Dairy;
import com.example.watering.investrecord.Info.Info_List6;
import com.example.watering.investrecord.R;

import java.text.DecimalFormat;
import java.util.ArrayList;
import java.util.Locale;

/**
 * Created by watering on 17. 11. 17.
 */

@SuppressWarnings("DefaultFileTemplate")
public class List6Adapter extends BaseAdapter {

    private final ArrayList<Info_List6> mData;
    private final LayoutInflater inflater;

    public List6Adapter(Context context, ArrayList<Info_List6> data) {
        mData = data;
        inflater = LayoutInflater.from(context);
    }

    @Override
    public int getCount() {
        return mData.size();
    }
    @Override
    public Object getItem(int position) {
        return mData.get(position);
    }
    @Override
    public long getItemId(int position) {
        return 0;
    }
    @Override
    public View getView(int position, View convertView, ViewGroup parent) {

        if(convertView == null) {
            convertView = inflater.inflate(R.layout.layout_list6,parent,false);
        }

        Info_Dairy dairy;
        TextView date = convertView.findViewById(R.id.textView_layout_list2_1);
        TextView principal = convertView.findViewById(R.id.textView_layout_list2_2);
        TextView evaluation = convertView.findViewById(R.id.textView_layout_list2_3);
        TextView rate = convertView.findViewById(R.id.textView_layout_list2_4);
        DecimalFormat df = new DecimalFormat("#,###");

        dairy = mData.get(position).getDairy();
        date.setText(String.valueOf(dairy.getDate()));
        principal.setText(df.format(dairy.getPrincipal()));
        rate.setText(String.format(Locale.getDefault(),"%.2f",dairy.getRate()));
        evaluation.setText(df.format(mData.get(position).getEvaluation()));

        return convertView;
    }
}

File: app/src/main/java/com/example/watering/investrecord/adapter/List6Adapter.kt
package com.example.watering.investrecord.adapter

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.BaseAdapter
import android.widget.TextView

import com.example.watering.investrecord.info.InfoList6
import com.example.watering.investrecord.R

import java.text.DecimalFormat
import java.util.ArrayList

/**
 * Created by watering on 17. 11. 17.
 */

class List6Adapter(context: Context, private val mData: ArrayList<InfoList6>) : BaseAdapter() {
    private val inflater: LayoutInflater = LayoutInflater.from(context)

    override fun getCount(): Int {
        return mData.size
    }

    override fun getItem(position: Int): Any {
        return mData[position]
    }

    override fun getItemId(position: Int): Long {
        return 0
    }

    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
        var convertView = convertView

        if (convertView == null) {
            convertView = inflater.inflate(R.layout.layout_list6, parent, false)
        }

        val txtMonth = convertView!!.findViewById<TextView>(R.id.textView_layout_list6_1)
        val txtIncome = convertView.findViewById<TextView>(R.id.textView_layout_list6_2)
        val txtSpend = convertView.findViewById<TextView>(R.id.textView_layout_list6_3)

        val list5 = mData[position]
        val df = DecimalFormat("#,###")

        txtMonth.text = list5.month
        txtIncome.text = df.format(list5.income.toLong())
        txtSpend.text = df.format(list5.spend.toLong())

        return convertView
    }
}
